src/main/resources/scripts/kill counter.sk

```diff
@@ -2,25 +2,20 @@
 # Counts kills per life, in total, and the highest kill streak.
 #
 
-# Defaults are useful here, as a player's kills would be '<none>' if he hasn't killed anything yet,
-# but with defaults defined those will be used in that case.
-variables:
-	{kill counter.%player%.kills_session} = 0
-	{kill counter.%player%.kills_session_max} = 0
-	{kill counter.%player%.kills_total} = 0
-
-on death:
+death:
```

Comment:
Start trigger with 'on' for clarity.

Reply:
Yeah, I should do that. I got used to not using the `on`.